NEWS.md

```diff
@@ -109,6 +109,8 @@ unit = "s")
 
 13. A relatively rare case of segfault when combining non-equi joins with `by=.EACHI` is now fixed, closes [#4388](https://github.com/Rdatatable/data.table/issues/4388).
 
+14. Non-equi joins now automatically set allow.cartesian to `TRUE`. Closes [4489](https://github.com/Rdatatable/data.table/issues/4489)
```

Comment:
It would be good to mention github handler of a reported of that issue. And if you wish also your, as contributor who provided PR.